From 457a874f98eab9bc0e51391ca756aa57fc8c986a Mon Sep 17 00:00:00 2001
From: Tony Cheng <tony.cheng@amd.com>
Date: Fri, 6 Jan 2017 15:55:47 -0500
Subject: [PATCH 2040/3082] drm/amd/display: enable clock gating and dchubp
 power gating

also refactor to clean reduce loc to achieve same logic

Change-Id: Ib2a02de2b49bf4b5c82108e59e8bd98f33a673aa
Signed-off-by: Tony Cheng <tony.cheng@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
Reviewed-by: Yongqiang Sun <yongqiang.sun@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dc.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dc.h b/drivers/gpu/drm/amd/display/dc/dc.h
index f53b413..e0143f9 100644
--- a/drivers/gpu/drm/amd/display/dc/dc.h
+++ b/drivers/gpu/drm/amd/display/dc/dc.h
@@ -145,7 +145,8 @@ struct dc_debug {
 	bool disable_stutter;
 	bool disable_dcc;
 	bool disable_dfs_bypass;
-	bool disable_power_gate;
+	bool disable_dpp_power_gate;
+	bool disable_hubp_power_gate;
 	bool disable_clock_gate;
 	bool disable_dmcu;
 	bool disable_color_module;
-- 
2.7.4

